ProjetSupaero2018/catkin_ws/src/opt_control/scripts/acado_client_example.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+import rospy
+from car_model.srv import OptControl
```

Comment:
Is car_model.srv still existing on your branch though ?

Reply:
Funny that it worked on my computer, there is no `car_model.srv.OptControl` in this branch, this code should fail. I'll change this line of code, thanks a lot.

We will have to think about removing the `car_model` package, when it's not used anymore.

Reply:
Maybe it worked because of generated files from previous catkin_make? Funny that they were not removed when you rebuilt though.

Reply:
You are absolutely right, nothing is removed from old builds, files with the same name are just replaced. That's the problem, I'll check that everything works fine after cleaning the build directory.